assets/sass/colors/_functional.scss

```diff
@@ -8,3 +8,12 @@ $aplus-info: #1d7792; // This is a darkened version of the bootstrap 3 default.
 $aplus-warning: #a6670e; // This is a darkened version of the bootstrap 3 default. Contrast must be >=4.5 with white.
 $aplus-danger: #a50000; // There should be a contrast ratio above 7 between this and white.
 $aplus-focus: #ffdd00; // This should be clearly different fron those defned above and the brand colours.
+
+// This colours may be used to represent the status of the grading.
+// The best color to use in order to have a good contrast level should be #000.
```

Comment:
What is this comment trying to say? That the contrast must be high against black? There is some English grammar mistake here.

Reply:
yes, I meant the color css property. But, maybe the comment is not needed since we can choose any other colour based on the contrast we want to achieve.